chain/x/zoracle/internal/types/request.go

```diff
@@ -1,32 +1,39 @@
 package types
 
 import (
-	"fmt"
-	"strings"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// Request is a type to store detail of request
+// Request is a type to store detail of request.
 type Request struct {
-	CodeHash    []byte `json:"codeHash"`
-	Params      []byte `json:"params"`
-	ReportEndAt uint64 `json:"reportEnd"`
+	OracleScriptID           int64            `json:"oracleScriptID"`
+	Calldata                 []byte           `json:"calldata"`
+	ValidatorList            []sdk.ValAddress `json:"validatorList"`
```

Comment:
Two things.
1. For `ValidatorList`, let's just call it `Validators` or `ActiveValidators`. Using word `List` when the type already tells you it's a list is bit redundant.
2. Do we need `SubmittedValidatorList`? I think we can just iterate on all reports, right? Maybe we just need `ReportCount`?

Reply:
Ok so we decide to keep both lists. Let's name it `RequestedValidators` and `ReceivedValidators`.